### PR TITLE
Input forms

### PR DESCRIPTION
**Pull request by kyam-mayden**
input forms created, logic informing database of changes and forms autofill info from DB

### CODE REVIEW

File: cmsController.php [hunk 1]
```diff
@@ -0,0 +1,171 @@
+<?php
+$db = new PDO('mysql:host=127.0.0.1; dbname=portfolioKyam', 'root');
+$db->setAttribute(PDO::ATTR_DEFAULT_FETCH_MODE, PDO::FETCH_ASSOC);
+
+/**
+ * Gets about sections name, content from Db and turns into an array
+ * @param $db to select from
```
Comment (JamesCullen2) [inline cmsController.php]: line breaks needed in doc blocks

File: cmsInput.php [hunk 2]
```diff
@@ -0,0 +1,159 @@
...
+                <input type="text" name="pfURL" placeholder="Enter new item URL here">
+                <br>
+                <label for="githubURL">github URL</label>
+                <input type="text" name="githubURL" placeholder="Enter new item repo URL here">
+                <br>
+                <label for="picSelect">Select picture</label>
+                <select name="picSelect">
+                    <?php echo makeImgDropDown($db) ?>
+                </select>
+
```
Comment (JamesCullen2) [inline cmsInput.php]: white space

File: tests/tests.php [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+    }
+
+    //malformed
+    public function testDropDownMalformed ()
+    {
+        $input1='title';
+        $this->expectException(TypeError::class);
+        makeDropDown($input1);
+    }
+
```
Comment (JamesCullen2) [inline tests/tests.php]: do you need this blank space?

**Review by JamesCullen2 — COMMENTED** [review]
Kyam, this looks very clean mate - super work. minor comments.

File: cmsController.php (hunk 1 above)
Comment (kyam-mayden) [reply cmsController.php]: Done

File: cmsInput.php (hunk 2 above)
Comment (kyam-mayden) [reply cmsInput.php]: done

File: tests/tests.php (hunk 3 above)
Comment (kyam-mayden) [reply tests/tests.php]: Resolved

**Review by JamesCullen2 — APPROVED** [review]

**Pull request merged**